connector/countconnector/README.md

```diff
@@ -0,0 +1,112 @@
+# Count Connector
+
+| Status                   |                                                           |
+|------------------------- |---------------------------------------------------------- |
+| Stability                | [in development]                                          |
+| Supported pipeline types | See [Supported Pipeline Types](#supported-pipeline-types) |
+| Distributions            | []                                                        |
+
+The `countconnector` can be used to count spans, data points, or log records.
```

Comment:
Nit: it's configured as `count` and is referenced to as the `count` connector in the connectors readme.

```suggestion
The `count` connector can be used to count spans, data points, or log records.
```